### DISCUSSION TITLE
Unable to put Lonsonho ZB-RGBCW in pairing mode

### DISCUSSION THREAD

**Discussion by denstu** [Koenkk/zigbee2mqtt#18317] [Devices]
Hello everybody.

I am unable to put Lonsonho ZB-RGBCW in pairing mode. I have 2 of them. After the first plugin the bulb pairs itself. But i am unable tu restart pairing. I followed the instructions from here:
https://www.zigbee2mqtt.io/devices/ZB-RGBCW.html

Turn on. Turn on/off, repeat 4-times. Nothing happens. I did it fast, i did it slower... Nothing. The light stays on, no blinking or pulsing.

Can anyone help?

**Comment by terry389** [comment]
I have exactly the same problem. I haven't been able to find a solution.

**Comment by wormvortex** [comment]
yup same. I managed to pair it one time then when I was moving my service to a new coordinator I've never been able to pair it since.

TBH they are crap bulbs the RGB colours are useless as they are so dim and now it doesn't pair either lol
